client/main.lua

```diff
@@ -178,177 +192,188 @@ local function CreateZone(type, number)
+        SetBlipColour(currentBlip, 3)
+        SetBlipRoute(currentBlip, true)
+        SetBlipRouteColour(currentBlip, 3)
     else
-        exports.qbx_core:Notify(Lang:t('success.payslip_time'), 'success', 5000)
-        if CurrentBlip ~= nil then
-            RemoveBlip(CurrentBlip)
+        exports.qbx_core:Notify(locale('success.payslip_time'), 'success', 5000)
+        if DoesBlipExist(currentBlip) then
+            RemoveBlip(currentBlip)
             ClearAllBlipRoutes()
-            CurrentBlip = nil
+            currentBlip = 0
         end
     end
 end
 
-local function CreateElements()
-    TruckVehBlip = AddBlipForCoord(sharedConfig.locations['vehicle'].coords.x, sharedConfig.locations['vehicle'].coords.y, sharedConfig.locations['vehicle'].coords.z)
-    SetBlipSprite(TruckVehBlip, 326)
-    SetBlipDisplay(TruckVehBlip, 4)
-    SetBlipScale(TruckVehBlip, 0.6)
-    SetBlipAsShortRange(TruckVehBlip, true)
-    SetBlipColour(TruckVehBlip, 5)
+local function createElements()
+    truckVehBlip = AddBlipForCoord(sharedConfig.locations.vehicle.coords.x, sharedConfig.locations.vehicle.coords.y, sharedConfig.locations.vehicle.coords.z)
+    SetBlipSprite(truckVehBlip, 326)
+    SetBlipDisplay(truckVehBlip, 4)
+    SetBlipScale(truckVehBlip, 0.6)
+    SetBlipAsShortRange(truckVehBlip, true)
+    SetBlipColour(truckVehBlip, 5)
     BeginTextCommandSetBlipName("STRING")
-    AddTextComponentSubstringPlayerName(sharedConfig.locations['vehicle'].label)
-    EndTextCommandSetBlipName(TruckVehBlip)
-
-    TruckerBlip = AddBlipForCoord(sharedConfig.locations['main'].coords.x, sharedConfig.locations['main'].coords.y, sharedConfig.locations['main'].coords.z)
-    SetBlipSprite(TruckerBlip, 479)
-    SetBlipDisplay(TruckerBlip, 4)
-    SetBlipScale(TruckerBlip, 0.6)
-    SetBlipAsShortRange(TruckerBlip, true)
-    SetBlipColour(TruckerBlip, 5)
+    AddTextComponentSubstringPlayerName(sharedConfig.locations.vehicle.label)
+    EndTextCommandSetBlipName(truckVehBlip)
+
+    truckerBlip = AddBlipForCoord(sharedConfig.locations.main.coords.x, sharedConfig.locations.main.coords.y, sharedConfig.locations.main.coords.z)
+    SetBlipSprite(truckerBlip, 479)
+    SetBlipDisplay(truckerBlip, 4)
+    SetBlipScale(truckerBlip, 0.6)
+    SetBlipAsShortRange(truckerBlip, true)
+    SetBlipColour(truckerBlip, 5)
     BeginTextCommandSetBlipName("STRING")
-    AddTextComponentSubstringPlayerName(sharedConfig.locations['main'].label)
-    EndTextCommandSetBlipName(TruckerBlip)
+    AddTextComponentSubstringPlayerName(sharedConfig.locations.main.label)
+    EndTextCommandSetBlipName(truckerBlip)
 
-    CreateZone('main')
-    CreateZone('vehicle')
+    createZone('main')
+    createZone('vehicle')
 end
 
-local function BackDoorsOpen(vehicle) -- This is hardcoded for the rumpo currently
+local function setBackDoorsOpen(vehicle) -- This is hardcoded for the rumpo currently
```

Comment:
```suggestion
local function areBackDoorsOpen(vehicle) -- This is hardcoded for the rumpo currently
```

We are not setting anything?